src/bindings/node/node.cc

```diff
@@ -97,7 +97,8 @@ class Actor {
   }
 
 private:
-  explicit Actor(const char* config) : actor(make_conf(config)), env_(nullptr), wrapper_(nullptr) {
+  explicit Actor(const char* config)
+      : actor(make_conf(config), true), env_(nullptr), wrapper_(nullptr) {
```

Comment:
auto-cleanup defaults to `false`. this is useful if you are some kind of service that wants explicit control over when you do cleanup. for example, if you want to be as fast as possible for your requesters, you probably want to call `cleanup` only after you've returned the response to them (this is how `prime_server` works). depending on the size of the request it shaves that extra little bit latency off the response time.